libraries/Microsoft.Bot.Builder.LanguageGeneration/Templates.cs

```diff
@@ -133,16 +134,17 @@ public class Templates : List<Template>
         public IList<string> Options { get; set; }
 
         /// <summary>
-        /// Gets a value indicating whether lG parser/checker/evaluate strict mode.
-        /// If strict mode is on, expression would throw exception instead of return
-        /// null or make the condition failed.
+        /// Gets a value LG parser/checker/evaluate options.
         /// </summary>
         /// <value>
-        /// A value indicating whether lG parser/checker/evaluate strict mode.
         /// If strict mode is on, expression would throw exception instead of return
         /// null or make the condition failed.
+        /// replaceNull works all the time. 
+        /// If strict=true then we throw on all conditions except for null value and we use replace text in case of null value.
+        /// lineBreakStyle = markdown means one line break will be translated to two line breaks, 
+        /// any other value will keep the same.
         /// </value>
-        public bool StrictMode => GetStrictModeFromOptions(Options);
+        public LGOptions LgOptions => GetLGOptions(Options);
```

Comment:
Also, we should accept an Options(or EvaluationOptions) in Evaluate method, and will override the options built-in files. 

Also options should be able to inherits from it's imports in a specific order we defined.